Formula/create-dmg.rb

```diff
@@ -0,0 +1,18 @@
+class CreateDmg < Formula
+  desc "Shell script to build fancy DMGs"
+  homepage "https://github.com/andreyvit/create-dmg"
+  url "https://github.com/andreyvit/create-dmg/archive/v1.0.0.4.tar.gz"
+  sha256 "bebc5fa200e55d8c792cb92699a0c4ba73e7dc7d4baf9814831f6963d0be589d"
+
+  def install
+    bin.install "create-dmg"
+    include.install "support/dmg-license.py"
```

Comment:
I don't believe `include` is a suitable directory for those… `include` is traditionally for header files (`.h`/`.hpp`).

Reply:
I know... Can `share` be used for those files? Looking at [Formula-Cookbook.md#variables-for-directory-locations](https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md#variables-for-directory-locations) did not find any way to add a custom folder.

Reply:
@fxcoudert will this be valid instead?

```rb
(prefix/"support").mkpath
(prefix/"support").install Dir["support/*"]
```

Reply:
Fixed

Reply:
@fxcoudert fixed 👍